woocommerce/google-listings-and-ads
# Attribute Mapping - Categories Frontend

### Changes proposed in this Pull Request:

Closes part of #1648 .

This PR implements the Category Tree Selector in the frontend.

### Screenshots:


https://user-images.githubusercontent.com/5908855/194890357-9d6b0343-891e-446c-9d90-291ca748330f.mov



### Detailed test instructions:


1. Create some categories in your Store
2. Go to Attribute Mapping and create some rules, see that the category selector shows your categories
3. Save the category and see that in the table, you can hover in the categories number and see the category names
4. Check that you can modify categories 
5. Create a new test category and assign it in a new rule
6. Delete the category afterwards
7. Check that the category appears as "Category ID %id% (deleted)
